hack/build/ctr/run.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# simple script to run cloudbuilds for ctr for each arch kind supports
+# NOTE: this is temporary, we only need to build ctr until our (tiny!)
+# --no-unpack patch is available in the standard package
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# cd to the repo root
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+
+# options
+PROJECT="${PROJECT:-bentheelder-kind-dev}"
+BUCKET="${BUCKET:-bentheelder-kind-dev/containerd}"
+# https://github.com/BenTheElder/containerd/tree/kind
+# https://github.com/containerd/containerd/pull/3259
+CONTAINERD_SOURCE="${CONTAINERD_SOURCE:-$(go env GOPATH)/src/github.com/containerd/containerd}"
+CONTAINERD_BRANCH="${CONTAINERD_BRANCH:-kind}"
+CTR_CLOUDBUILD="${REPO_ROOT}/hack/build/ctr/cloudbuild.yaml"
+
+# make sure we submit the build to the righr project
```

Comment:
`right`

Reply:
not sure why check-spelling doesn't catch some of these.
feel free to ignore.

/lgtm
/hold

Reply:
fixed, went ahead and removed the .gcloudignore bit as well since we no longer need that